internal/pkg/deploy/cloudformation/stack/lb_web_service_integration_test.go

```diff
@@ -61,6 +64,13 @@ func TestLoadBalancedWebService_Template(t *testing.T) {
 	manifestBytes, err := ioutil.ReadFile(path)
 	require.NoError(t, err)
 	for name, tc := range testCases {
+		ctrl := gomock.NewController(t)
```

Comment:
I'm just wondering instead of mock addon whether we should use `addons.New()` in the local integration test so that it simulates the real work flow as much as possible?

Reply:
changed in https://github.com/aws/copilot-cli/pull/3820/commits/ea721cc75996dbf65858faa8b81e6da2e53878a4!